src/SSHDebugPS/SSHDebugPS.csproj

```diff
@@ -68,7 +68,7 @@
     <Reference Include="Microsoft.VisualStudio.OLE.Interop, Version=7.1.40304.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a">
       <Private>False</Private>
     </Reference>
```

Comment:
I think you removed the only reference to this in this project. you can probably remove this now